dev-docs/bundle-rfc.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2019-03-12
+- RFC PR: [#408](https://github.com/uber/xviz/pull/408)
+- XVIZ Issue: [#250](https://github.com/uber/xviz/issues/250)
+
+# Summary
+
+We make the PNG and GIF of XVIZ. By making a `xviz/bundle` type that combines a `metadata` field
+with a list of `state_update` messages. This has all the information to view an XVIZ scene in on
+file. It's small enough to need no index, and can be stored in either JSON or GLB format.
```

Comment:
The index part is curious as I do not know what it implies.  Size or number of frames?  Some beta partners where output data at a high HZ which was causing problems not related to the size, but the fact that they has far more "state_updates" than was normal. 

Can we tight up the intention behind why we are not putting in an index?

Reply:
I think the presence of an index should be optional.  I don't think the bundle format should be limited to some concept of small.

Reply:
The problem with an index is that it's really only needed if you don't want to read all of the file at once.  Just like with GIF and PNG they don't really support this with their animations.   We have to design a format for that case in specific instead of just a simple message we can pass around.